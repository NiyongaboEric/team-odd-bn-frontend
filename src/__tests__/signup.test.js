import React from 'react';
import { shallow } from 'enzyme';
import Signup from '../containers/user/signup/signup';

let wrapper;

describe('Signup testing', () => {
  wrapper = shallow(<Signup />);
  it('Should render class', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
