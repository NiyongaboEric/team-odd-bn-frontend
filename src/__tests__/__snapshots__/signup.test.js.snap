// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup testing Should render class 1`] = `ShallowWrapper {}`;
